Cargo.toml

```diff
@@ -29,8 +29,10 @@ sha2 = "0.9"
 thiserror = "1"
 
 [dev-dependencies]
+elements-consensus = { path = "../rust-elements-consensus" }
```

Comment:
I had to update `elements-consensus` to `rust-elements:0.18.0` since this branch is now rebased onto #37.

Reply:
Just waiting for https://github.com/comit-network/rust-elements-consensus/pull/4 to be merged now. Then I'll update this.